core/src/main/java/org/elasticsearch/index/query/InnerHitBuilder.java

```diff
@@ -47,7 +47,7 @@
 
 import static org.elasticsearch.common.xcontent.XContentParser.Token.END_OBJECT;
 
-public final class InnerHitBuilder extends ToXContentToBytes implements Writeable {
+public final class InnerHitBuilder implements Writeable, ToXContentObject {
```

Comment:
should we add toString impl to this class now that it doesn't inherit anymore from ToXContentToBytes?